components/primary_button.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { StyleSheet, Text, View, Button } from 'react-native';
+
+
+
+export default class primary_button extends React.Component {
+  _onPressButton() {
+    Alert.alert('You tapped the button!')
+  }
+
+  render() {
+    return (
+      <View style={styles.container}>
+         <View style={styles.RectangleShapeViewOne}></View>
+         <View style={styles.buttonContainer}>
+          <Button
+            onPress={this._onPressButton}
+            title="Press Me"
+            color = '#fff'
+          
+          />
+            </View>
+      </View>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
+  container: {
```

Comment:
Remove the `container` style - you should only be applying styles to the component itself since it should be placed on numerous pages